.snippets/webservers/Caddyfile-nossl

```diff
@@ -0,0 +1,48 @@
+{
+    servers :80 {
+        timeouts {
+            read_body 120s
+        }
+    }
+}
+
+<domain>:80 {
```

Comment:
Why is this highlighted?
![Screenshot_2023-11-28-13-28-30-289_com.android.chrome-edit.jpg](https://github.com/pterodactyl/documentation/assets/71085027/5ed22e07-fcc6-412a-a560-ea40f1950699)

Reply:
Because it is important change to switch off HTTPS. It is required by Caddy to do so.

Reply:
The highlighting means the line that should be edited, yet the line I mentioned shouldn't be...? It is in the non-SSL config